eva/eva.yml

```diff
@@ -15,7 +15,10 @@ executor:
   gpus: {'127.0.0.1': [0]}
 storage:
   engine: "eva.storage.petastorm_storage_engine.PetastormStorageEngine"
+  video_engine: "eva.storage.video_storage_engine.VideoStorageEngine"
+  video_engine_version: 0
```

Comment:
What is the use for this version?

Reply:
Not sure how this is supposed to work. Right now in `_get_video_file_path`, we compare the version in the metadata and version in the yml file, and reject the request if the version in the metadata is larger then version in the yml file. What you suggested is that say later we have `video_engine_version: 1`, and for metadata having `version 0`, we will still use the version 0 storage engine.

Reply:
One way is to have a `switch case` in StorageEngine.read and StorageEngine.write.

Reply:
So the idea is that later as we have another version, we handle them differently.

Reply:
yml file keeps track of the latest version number